ecosystem/indexer-grpc/indexer-grpc-fullnode/src/runtime.rs

```diff
@@ -58,9 +62,25 @@ pub fn bootstrap(
             service_context: service_context.clone(),
         };
         let localnet_data_server = LocalnetDataService { service_context };
-        let mut tonic_server = Server::builder()
+
+        let reflection_service = tonic_reflection::server::Builder::configure()
+            // Note: It is critical that the file descriptor set is registered for every
+            // file that the top level API proto depends on recursively. If you don't,
+            // compilation will still succeed but reflection will fail at runtime.
+            //
+            // TODO: Add a test for this / something in build.rs, this is a big footgun.
+            .register_encoded_file_descriptor_set(INDEXER_V1_FILE_DESCRIPTOR_SET)
+            .register_encoded_file_descriptor_set(TRANSACTION_V1_TESTING_FILE_DESCRIPTOR_SET)
+            .register_encoded_file_descriptor_set(UTIL_TIMESTAMP_FILE_DESCRIPTOR_SET)
+            .build()
+            .expect("Failed to build reflection service");
+
+        let reflection_service_clone = reflection_service.clone();
+
+        let tonic_server = Server::builder()
             .http2_keepalive_interval(Some(std::time::Duration::from_secs(60)))
-            .http2_keepalive_timeout(Some(std::time::Duration::from_secs(5)));
+            .http2_keepalive_timeout(Some(std::time::Duration::from_secs(5)))
+            .add_service(reflection_service_clone);
```

Comment:
I'm confused lol, why are we need this now and how come we didn't need it before?

Reply:
This is the new service that Larry just added recently so we didn't really have anything before. Without this you can't use grpcurl without "importing" all the proto files. Just a nice UX win, we have this on some of our other grpc servers.